classes/linux-qcom-bootimg.bbclass

```diff
@@ -1,3 +1,9 @@
+#
+# Copyright (c) 2021 Linaro
```

Comment:
This is questionable. Code originates from `linux-qcom-bootimg.inc` which should be `2015-2021,2023 (c) Linaro`. The bbclass has been further modified by Quic in 2023-2024 and by Snap in 2024.
So, this simple class brings in usual question of what is copyrightable and whether we should track all contributions as copyright headers.

Reply:
When reviewing the changes I decided to only add copyright for the major changes, but there is indeed a few more entries when reviewing the entire linux-qcom-bootimg.inc history, will add to avoid issues.

Reply:
Added for all contributors to avoid any concern.